specification/search/data-plane/Azure.Search/preview/2024-05-01-Preview/searchindex.json

```diff
@@ -1941,6 +2075,95 @@
             "value": "text",
             "name": "Text",
             "description": "Vector query where a text value that needs to be vectorized is provided."
+          },
+          {
+            "value": "imageUrl",
+            "name": "ImageUrl",
+            "description": "Vector query where an url that represents an image value that needs to be vectorized is provided."
+          },
+          {
+            "value": "imageBinary",
+            "name": "ImageBinary",
+            "description": "Vector query where a base 64 encoded binary of an image that needs to be vectorized is provided."
+          }
+        ]
+      },
+      "description": "The kind of vector query being performed."
+    },
+    "VectorThreshold": {
+      "type": "object",
+      "discriminator": "kind",
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/VectorQueryKind",
+          "description": "The kind of vector query being performed.",
+          "x-nullable": false
+        }
+      },
+      "required": [
+        "kind"
+      ],
+      "description": "The threshold used for vector queries."
+    },
+    "VectorSimilarityThreshold": {
+      "type": "object",
+      "x-ms-discriminator-value": "vectorSimilarity",
+      "allOf": [
+        {
+          "$ref": "#/definitions/VectorThreshold"
+        }
+      ],
+      "properties": {
+        "value": {
+          "type": "number",
+          "format": "double",
+          "description": "The threshold will filter based on the similarity metric value. Note this is the canonical definition of similarity metric, not the 'distance' version. The threshold direction (larger or smaller) will be chosen automatically according to the metric used by the field."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The results of the vector query will be filtered based on the vector similarity metric. Note this is the canonical definition of similarity metric, not the 'distance' version. The threshold direction (larger or smaller) will be chosen automatically according to the metric used by the field."
+    },
+    "SearchScoreThreshold": {
+      "type": "object",
+      "x-ms-discriminator-value": "searchScore",
+      "allOf": [
+        {
+          "$ref": "#/definitions/VectorThreshold"
+        }
+      ],
+      "properties": {
+        "value": {
+          "type": "number",
+          "format": "double",
+          "description": "The threshold will filter based on the '@search.score' value. Note this is the @search.score returned as part of the search response. The threshold direction will be chosen for higher @search.score."
```

Comment:
Range? Document all ranges if not otherwise obvious.

Reply:
Answered in different comment thread :)